source/_integrations/legrand_homepluscontrol.markdown

```diff
@@ -0,0 +1,103 @@
+---
+title: Legrand Home+ Control
+description: Instructions on how to integrate Legrand Home+ Control into Home Assistant.
+ha_category:
+  - Switch
+ha_release: '2021.03'
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@chemaaa'
+ha_config_flow: true
+ha_domain: homepluscontrol
+---
+
+The `Home+ Control` integration platform allows you to control a range of Legrand in-wall switches and power outlets that have smarthome functionality thanks to their "with Netatmo" capabilities. 
+
+This integration works against the `Home+ Control` API, which is one of the many APIs offered through the *Works with Legrand* program (https://developer.legrand.com/). The API is capable of managing "Legrand/Btcino with Netatmo" devices, such as light switches, power outlets and rolling shutters.
+
+The devices that this API can manage are offered in different designs across different countries. The details of these can be found in the following page: https://developer.legrand.com/solutions/wiring-devices-with-netatmo/
+
+Legrand offers the Home+ Control smartphone app to manage these devices. You can find this app in the Google Play and Apple App stores.
+
+This `Home+ Control` integration for Home Assistant currently has support for the following devices: 
+- Light switches
+- Power outlets
+
+In both cases, the devices are modelled as simple on/off switches within Home Assistant.
+
+This integration has been tested to work with the following range of Legrand products:
+- Valena Next™ with Netatmo
+
+{% include integrations/config_flow.md %}
+
+## Authentication
+
+Before you are able to configure the Legrand `Home + Control` integration into Home Assistant, you must register with the *Works with Legrand* platform.
+
+These Legrand APIs rely on Oauth2 authentication, so you must follow these steps to obtain the necessary authentication parameters:
+
+1) Register an account at https://developer.legrand.com/
+2) Create a subscription to the *Starter Kit* (currently the only subscription available) and this will generate your SUBSCRIPTION_KEY.
+3) Register an Application, where you will have to define a name, a redirect URL and the scopes of your application (for simplicity you can mark all scopes). 
```

Comment:
```suggestion
1. Register an account at <https://developer.legrand.com>.
2. Create a subscription to the *Starter Kit* (currently the only subscription available) and this will generate your `SUBSCRIPTION_KEY`.
3. Register an application, where you will have to define a name, a redirect URL and the scopes of your application (for simplicity you can mark all scopes).
```

> (for simplicity you can mark all scopes)

That sounds like a general bad practice. Also, the right scopes aren't documented either.